pom.xml

```diff
@@ -632,8 +632,8 @@
 
                 <configuration>
                     <argLine>-Xmx3072m</argLine>
-                    <rerunFailingTestsCount>3</rerunFailingTestsCount>
-                    <forkCount>3</forkCount>
+                    <rerunFailingTestsCount>5</rerunFailingTestsCount>
+                    <forkCount>5</forkCount>
```

Comment:
Is the `forkCount` change needed? would we have enough memory with the 3Gb allocated in L653 if we now have 5 threads running tests? I think we'd be ok just changing the `rerunFailingTestsCount` attribute to `5`.

Reply:
okay... i wasn't sure about this change... but i will change forkCount back to 3